styles/index.module.css

```diff
@@ -0,0 +1,39 @@
+.container{
+    display: grid;
+    height: 100vh;
+    grid-template-columns: 1fr;
+    grid-template-rows: 0.2fr 2fr 0.2fr;
+    grid-gap: 0.2rem;
+    grid-template-areas: 
+        "header"
+        "main"
+        "footer";
+}
+.header{
+    background-color: #a7ffeb;
+    padding: 5px;
+    grid-area: header;
+}
+.main{
+    background-color: #73ffba;
+    padding: 5px;
+    grid-area: main;
+}
+.footer{
+    background-color: #18ffff;
+    padding: 5px;
+    grid-area: footer;
+}
+@media only screen and (max-width:550px){
```

Comment:
is there any better approach than using media queries to create a responsive layout? Have you checked any npm library that might help?